### PR TITLE
R packages crew and crew.cluster, along with prerequisites.  From conda_r_skeleton_helper

### PR DESCRIPTION
**Pull request by brendanf**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by brendanf** [comment]
@conda-forge/help-r
The Windows error is
[`ERROR (r-getip,lib/R/library/getip/libs/x64/getip.dll): $RPATH/R.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?`](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=843842&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=7c0f8eae-6d6f-51bf-636f-73a1a7fb1bc4&l=1039).

I'm far from an expert on getting things to build in Windows.  I would expect that `R.dll` should be packaged by `r-base`, but it somehow isn't being found in this check (although the actual `R CMD install` seems to have worked fine). I can't see that `getip` is doing anything unconventional with library paths.  Anyone have any advice?

File: recipes/r-mirai/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    - r-base
+    - r-nanonext >=0.10.4
+
+test:
+  commands:
+    - $R -e "library('mirai')"           # [not win]
+    - "\"%R%\" -e \"library('mirai')\""  # [win]
+
+about:
+  home: https://shikokuchuo.net/mirai/, https://github.com/shikokuchuo/mirai/
```
Comment (mfansler) [inline recipes/r-mirai/meta.yaml:40]: split urls:

```suggestion
  home: https://shikokuchuo.net/mirai/
  dev_url: https://github.com/shikokuchuo/mirai/
```

File: recipes/r-crew/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    - r-tibble
+    - r-tidyselect
+
+test:
+  commands:
+    - $R -e "library('crew')"           # [not win]
+    - "\"%R%\" -e \"library('crew')\""  # [win]
+
+about:
+  home: https://wlandau.github.io/crew/, https://github.com/wlandau/crew
```
Comment (mfansler) [inline recipes/r-crew/meta.yaml:58]: split urls

```suggestion
  home: https://wlandau.github.io/crew/
  dev_url: https://github.com/wlandau/crew
```

File: recipes/r-crew.cluster/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    - r-lifecycle
+    - r-rlang
+
+test:
+  commands:
+    - $R -e "library('crew.cluster')"           # [not win]
+    - "\"%R%\" -e \"library('crew.cluster')\""  # [win]
+
+about:
+  home: https://wlandau.github.io/crew.cluster/, https://github.com/wlandau/crew.cluster
```
Comment (mfansler) [inline recipes/r-crew.cluster/meta.yaml:46]: split urls

```suggestion
  home: https://wlandau.github.io/crew.cluster/
  dev_url: https://github.com/wlandau/crew.cluster
```

**Review by mfansler — CHANGES_REQUESTED** [review]
LGTM - just a minor touch up, and we need to wait for `r-getip` and `r-nanonext` to get generated.

**Comment by brendanf** [comment]
@conda-forge-admin please rerender

**Comment by brendanf** [comment]
@conda-forge-admin, please restart ci

**Comment by brendanf** [comment]
@conda-forge/help-r Now that the prerequisites are available, this builds on Linux and OSX.  Windows is failing due to #25319

**Review by mfansler — APPROVED** [review]

**Comment by mfansler** [comment]
Thanks for adding these! ❤️ 

Note that since these have interdependencies (`crew.cluster -> crew -> mirai`), the CI on the feedstocks will likely fail in the first round and require manual restarts. I'll try to keep an eye out, but you can get view the CI jobs by clicking the indicator (red "X" or green check) next to commit on the feedstock pages. Options to "Re-run" failed jobs should be there.

**Pull request merged**